src/broadcast.jl

```diff
@@ -62,15 +62,20 @@ function common_chunks(s, args...)
     all(ar -> isa(eachchunk(ar), GridChunks), chunkedarrays) ||
         error("Currently only chunks of type GridChunks can be merged by broadcast")
     if isempty(chunkedarrays)
-        totalsize = sum(sizeof ∘ eltype, args)
-        return estimate_chunksize(s, totalsize)
```

Comment:
I think this function does actually the wrong thing. Can we just call into this method https://github.com/JuliaIO/DiskArrays.jl/blob/877fd67b3374089ed30a4c937c2db97b4af22dd6/src/chunks.jl#L441 and have the non-isbits types covered?

Reply:
@meggart I honestly don't know what you mean here. Can you just edit it into the code as a suggestion thats usually clearer.

This is breaking things in Rasters.jl and this comment is blocking from merging a fix that works

Reply:
Can't suggest for a deleted line

Reply:
No you but you can suggest on the other side?